tests/tests/test-governance.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from "chai";
+import Keyring from "@polkadot/keyring";
+import {
+  ALITH_PRIV_KEY,
+  DOROTHY,
+  DOROTHY_PRIV_KEY,
+  ETHAN,
+  ETHAN_PRIVKEY,
+  PROPOSAL_AMOUNT,
+  VOTE_AMOUNT,
+} from "../util/constants";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import { execFromTwoThirdsOfCouncil, execFromAllMembersOfTechCommittee } from "../util/governance";
+
+const keyring = new Keyring({ type: "ethereum" });
+
+let alith;
+let dorothy;
+let ethan;
+
+describeDevMoonbeam("Governance", (context) => {
+  before("Create accounts", async () => {
+    alith = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+    dorothy = await keyring.addFromUri(DOROTHY_PRIV_KEY, null, "ethereum");
```

Comment:
You are not using Dorothy nor Ethan AFAICT. You can remove them

Reply:
Removed